tests/test_generics.py

```diff
@@ -2299,3 +2301,65 @@ class Model(BaseModel, Generic[T]):
         'title': 'Model[test_generic_intenum_bound.<locals>.MyEnum]',
         'type': 'object',
     }
+
+
+@pytest.mark.skipif(sys.version_info < (3, 11), reason='requires python 3.11 or higher')
+@pytest.mark.xfail(reason='TODO: Variadic generic parametrization is not supported yet')
+def test_variadic_generic_init():
+    class ComponentModel(BaseModel):
+        pass
+
+    class Wrench(ComponentModel):
+        pass
+
+    class Screwdriver(ComponentModel):
+        pass
+
+    ComponentVar = TypeVar('ComponentVar', bound=ComponentModel)
+    NumberOfComponents = TypeVarTuple('NumberOfComponents')
+
+    class VariadicToolbox(BaseModel, Generic[ComponentVar, *NumberOfComponents]):
```

Comment:
Please use [Unpack](https://peps.python.org/pep-0646/#unpack-for-backwards-compatibility) instead of the new `*TypeVarTuple` syntax so that we don't get syntax errors when attempting to parse the tests in older versions of python